### ISSUE TITLE
Improve the color contrast according to WCAG 2.2 Level AAA

### ISSUE THREAD

**Issue by LabinatorSolutions** [charmbracelet/glow#809]
The default color contrast is low. For minimum accessibility compliance, it should be at least **7:1** according to **WCAG 2.2 Level AAA**. For optimal readability across all visual conditions, aim for a contrast ratio of at least **9:1**.

You can check and verify the color accessibility while generating ready-made accessible palettes at:
[https://tools.labinator.com/color-accessibility](https://tools.labinator.com/color-accessibility)
